src/dsp_tools/utils/logger_config.py

```diff
@@ -3,6 +3,7 @@
 from loguru import logger
 
 logger_savepath = (Path.home() / ".dsp-tools" / "logging.log").absolute()
+warnings_savepath = Path("warnings.log")
```

Comment:
These 2 should probably be constants, in upper case. What do you think?

Reply:
I am fine with either.